src/twisted/internet/interfaces.py

```diff
@@ -1364,7 +1371,7 @@ class IReactorInThreads(Interface):
     @since: 15.4
     """
 
-    def callInThread(callable, *args, **kwargs):
+    def callInThread(callableObj, *args, **kwargs):
```

Comment:
I think is best to have a separate PR:

```
$ grep -r "def callInThread(" src/
src/twisted/python/threadpool.py:    def callInThread(self, func, *args, **kw):
src/twisted/web/test/test_wsgi.py:    def callInThread(self, f, *a, **kw):
src/twisted/internet/test/test_base.py:    def callInThread(self, callable, *args, **kwargs):
src/twisted/internet/interfaces.py:    def callInThread(callable, *args, **kwargs):
src/twisted/internet/base.py:        def callInThread(self, _callable, *args, **kwargs):
```

Reply:
I've moved this to: https://github.com/twisted/twisted/pull/1325